.github/workflows/github-projects.yml

```
@@ -0,0 +1,20 @@
+name: "Add To GitHub projects"
```

```suggestion
name: "Add To CCF Current Horizon GitHub project"
```

Just a suggestion to make it slightly more precise.